dataparser/index/index.py

```diff
@@ -0,0 +1,289 @@
+import logging
+import re
+import multiprocessing
+import xml.etree.ElementTree as ET
+from os import path, listdir
+from json import load as json_load
+from json import dump as json_dump
+from collections import namedtuple
+from parser_utils.file_formatter import rf_table_name, lib_table_name
+from parser_utils.util import get_index_name
+from queue.queue import ParsingQueue
+from data_parser.data_parser import DataParser
+from db_json_settings import DBJsonSetting
+from queue.finder import finder
+
+logging.basicConfig(

```

Comment:
I see this repeated in several places. Maybe the dict with config can be moved to some place where "commons" live and then used as needed.